draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -386,7 +385,7 @@ social relations create webs of human connection. This can be made rigorous and
 verifiable with self-sovereign identities held by each person, documented
 digitally by mutual verification asserted cryptographically through a secure
 digital medium. Creating this type of network to socially validate an identity
-is why this aspect of the technology is called a **web of trust.**
+is why this aspect of the technology has been called a **web of trust.**
```

Comment:
This is gone following @shannona's edit. Take a look and see how you like the current rev.